src/main/js/isd.js

```diff
@@ -309,7 +308,7 @@
         for (var qnameb in isd_element.styleAttrs) {
             var da = imscStyles.byQName[qnameb];
 
-            if (da.applies.indexOf(isd_element.kind) === -1) {
+            if (da !== undefined && da.applies !== undefined && da.applies.indexOf(isd_element.kind) === -1) {
```

Comment:
`da` should never be undefined since isd element styles are from the same set as the document element styles